meson.build

```diff
@@ -223,6 +223,12 @@ else
   glfwDep = compiler.find_library('glfw', required: false)
 endif
 
+# cpp-httplib
+httpLibAvailable=compiler.has_header('httplib.h', required: false)
+
+# nlohmann_json
+jsonDep = dependency('nlohmann_json', required: false)
```

Comment:
we have own copy of this library in `libosmscout-client/include/osmscoutclient/json`, I would consider to link MCPServer with libosmscout-client library to re-use asynchronous API ;-)

Reply:
hmm, but httplib have no synchronous api, right?